manifests/params.pp

```diff
@@ -1,9 +1,10 @@
 # Set default parameters based on distribution release
 class nfs::params {
   $portmap = $::operatingsystem? {
-    'Debian' => $::lsbdistcodename? {
-      'Wheezy' => 'rpcbind',
-      default  => 'portmap',
+    'Debian' => versioncmp($::operatingsystemmajrelease, 7)? {
```

Comment:
`versioncmp` returns a number, not a string. This works in Puppet 3 because of very loose typing, but not in Puppet 4.

Use this instead:

```puppet
'Debian' => versioncmp($::operatingsystemmajrelease, 7) >= 0 ? {
  true => 'rpcbind',
  false => 'portmap',
}
```